gsk4/src/render_replay.rs

```diff
@@ -0,0 +1,259 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::{ffi, RenderNode};
+use glib::{prelude::*, translate::*};
+use std::boxed::Box as Box_;
+
+#[repr(C)]
+pub struct RenderReplay(std::ptr::NonNull<ffi::GskRenderPlay>);
+
+impl Drop for RenderReplay {
+    fn drop(&mut self) {
+        unsafe {
+            ffi::gsk_render_replay_free(self.ptr.as_ptr());
+        }
+    }
+}
+
+impl RenderReplay {
+    #[doc(alias = "gsk_render_replay_default")]
+    #[allow(clippy::should_implement_trait)]
+    pub fn default(&self, node: impl AsRef<RenderNode>) -> Option<RenderNode> {
+        unsafe {
+            from_glib_full(ffi::gsk_render_replay_default(
+                self.to_glib_none_mut().0,
```

Comment:
You no longer have access to glib::translate::*, you should do self.ptr.as_ptr() for getters and as_mut_ptr for setters (those would require &mut self)

Reply:
I don't think https://docs.gtk.org/gsk4/struct.RenderReplay.html has any getter and I am not sure if the non `set_` methods should be mut.